### PR TITLE
feat: supporting up_status_codes

### PR DESCRIPTION
**Pull request by jeansebastienh**
Pre requisite: https://github.com/Bonial-International-GmbH/site24x7-go/pull/32

### CODE REVIEW

**Comment by martinohmann** [comment]
Hey @jeansebastienh. You can rebase on the current master to continue working on this. site24x7-go was updated in https://github.com/Bonial-International-GmbH/terraform-provider-site24x7/pull/24 to the latest version that includes https://github.com/Bonial-International-GmbH/site24x7-go/pull/32.

**Comment by jeansebastienh** [comment]
Oups I missed the notification.

I just rebased with origin/master :)

Thanks @martinohmann

File: site24x7/websitemonitor.go [hunk 1]
```diff
@@ -283,6 +288,7 @@ func resourceDataToWebsiteMonitor(d *schema.ResourceData, client site24x7.Client
 		UserGroupIDs:          userGroupIDs,
 		ActionIDs:             actionRefs,
 		UseNameServer:         d.Get("use_name_server").(bool),
+		UpStatusCodes:          d.Get("up_status_codes").(string),
```
Comment (martinohmann) [inline site24x7/websitemonitor.go:291]: The indentation looks weird here, can you `go fmt` the file?

**Review by martinohmann — CHANGES_REQUESTED** [review]

File: site24x7/websitemonitor.go (hunk 1 above)
Comment (jeansebastienh) [reply site24x7/websitemonitor.go]: Yes sure :+1:

**Review by martinohmann — APPROVED** [review]
This looks good now, thanks for the contribution!

**Pull request merged**